### PR TITLE
Update code in the tutorial

### PR DESCRIPTION
**Pull request by fadils**
This update to reflect a more consistent code
between the website and the `relay-treasurehunt` example.

### CODE REVIEW

**Comment by facebook-github-bot-7** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by facebook-github-bot-5** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

**Comment by wincent** [comment]
Thanks!

**Comment by wincent** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-7** [comment]
I cannot do that @wincent, because @fadils does not have an active CLA on file.

File: docs/QuickStart-Tutorial.md [hunk 1]
```diff
@@ -28,43 +28,43 @@ We need a place to hide our treasure, a way to check hiding spots for treasure,
...
-  var hidingSpot = getHidingSpot(id);
-  hidingSpot.hasBeenChecked = true;
-}
-export function getHidingSpot(id) {
-  return hidingSpots.find(hs => hs.id === id);
-}
-export function getGame() { return game; }
-export function getHidingSpots() { return hidingSpots; }
-export function getTurnsRemaining() { return turnsRemaining; }
+ // Model types
```
Comment (josephsavona) [inline docs/QuickStart-Tutorial.md]: nit: can you remove the leading space from all of these lines?

File: docs/QuickStart-Tutorial.md [hunk 2]
```diff
@@ -148,14 +148,14 @@ var gameType = new GraphQLObjectType({
...
 
-var hidingSpotType = new GraphQLObjectType({
+const hidingSpotType = new GraphQLObjectType({
   name: 'HidingSpot',
   description: 'A place where you might find treasure',
   fields: () => ({
     id: globalIdField('HidingSpot'),
     hasBeenChecked: {
       type: GraphQLBoolean,
-      description: 'True if this spot has already been checked for treasure',
```
Comment (sampepose) [inline docs/QuickStart-Tutorial.md]: can you fix this typo please?

Comment (fadils) [inline docs/QuickStart-Tutorial.md]: Sorry for this. I was doing a copy-paste from the example. https://github.com/relayjs/relay-examples/blob/master/relay-treasurehunt/data/schema.js#L91

**Comment by sampepose** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-8** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/536004673251149/int_phab) to review internal test results.

**Pull request closed without merging**